libstuff/libstuff.cpp

```diff
@@ -2615,6 +2615,10 @@ int SQuery(sqlite3* db, const char* e, const string& sql, SQResult& result, int6
     if ((int64_t)elapsed > warnThreshold) {
         // This code removing authTokens is a quick fix and should be removed once https://github.com/Expensify/Expensify/issues/144185 is done.
         pcrecpp::RE("\"authToken\":\"[0-9A-F]{400,1024}\"").GlobalReplace("\"authToken\":<REDACTED>", &sqlToLog);
+
+        // We remove anything inside "html" because it's most likely sensitive data that should not be seen in logs
+        pcrecpp::RE("\"html\":\".*\"").GlobalReplace("\"html\":<REDACTED>", &sqlToLog);
```

Comment:
`.*` tends to be greedy matching so I am not sure if you will end up redacting everything or will stop at the first `"` that it sees.

Reply:
![image](https://user-images.githubusercontent.com/6474442/215162837-11089a8b-2d07-4911-84ec-a722ffabd7d3.png)

Reply:
Tested by sending the message `test with "quotes"`:
```
2023-01-27T18:13:25.253865+00:00 expensidev2004 bedrock: 4hpwda florent@expensifail.com (libstuff.cpp:2654) SQuery [socket7] [warn] 'read/write transaction', query failed with error #19 (UNIQUE constraint failed: reportActions.reportActionID): INSERT INTO reportActions ( reportActionID, created, reportID, accountID, action, message ) VALUES ( 6311291693237444632, '2023-01-27 18:13:25.253', 345, 30, 'ADDCOMMENT', REPLACE ('{"html":<REDACTED>}', '%reportActionID%', 6311291693237444632));
```

Reply:
We chatted about this on Slack. Summary: Having an object as '{"html":"blabla", "data":"blabla", "account":"blabla"}' will turn the whole thing as `'{"html":<REDACTED>}' but it is not important as having sensitive info is worse and if someone needs the whole object for debugging purposes, they can query the DB

Reply:
Discussed in Slack that we'd rather redact the whole JSON rather than leave sensitive info in logs